lib/commons/forms/is-disabled.js

```diff
@@ -0,0 +1,40 @@
+const disabledNodeNames = ['fieldset', 'button', 'select', 'input', 'textarea'];
+
+/**
+ * Determines if an element disabled, or part of a disabled element
+ *
+ * IMPORANT: This method is fairly loose. There are significant differences in browsers of when
+ * they'll announce a thing disabled. This tells us if any accessibility supported browser
+ * identifies an element as disabled, but not if all of them do.
+ *
+ * @method isDisabled
+ * @memberof axe.commons.forms
+ * @param {VirtualNode} virtualNode
+ * @return {boolean} whether or not the element is disabled in some way
+ */
+function isDisabled(virtualNode) {
+	let disabledState = virtualNode._isDisabled;
+	if (typeof disabled === 'boolean') {
```

Comment:
Did you mean `disabledState`?

```suggestion
	if (typeof disabledState === 'boolean') {
```